lib/SearchAndSortPane/SearchAndSortPane.js

```diff
@@ -75,13 +75,19 @@ export class SearchAndSortPane extends Component {
     resourceName: PropTypes.string.isRequired,
     resultCountIncrement: PropTypes.number.isRequired,
     initialResultCount: PropTypes.number.isRequired,
+    hasSearchForm: PropTypes.bool,
```

Comment:
I'm not sure that we should have the ability to hide search. If the search is not needed in some cases, it seems, it might be more useful to split search and sort functionality and just to combine them in the SearchAndSort component. 
Or if we are going to add the search in the future (we can clarify this requirement), we can disable (not hide) it for all cases because the search is not supported for now.

What do you think about it?

Reply:
I think it makes sense to clarify this in the future and if the search is not applicable for choose a job profile scenario then maybe it makes sense to get rid of it for it. I suggest proceeding as is in this case as search initially was not present on the mockup. As I see from data-import experience introducing search for choose profile page would require considerable amount of effort.

Reply:
So, in this case, I'm not sure that SearchAndSortPane should be used because the only Sort part from SearchAndSortPane should be used.

Reply:
With the prop which I provided, I see it works just fine. In this case, the search is needed and in another, it is not. Building a new component or doing the refactoring for splitting SearchAndSortPane is beyond the estimate for this story in my opinion.

Reply:
If the search is displayed in settings (Settings -> Data export -> Job Profiles) so it means that list can be long. Are there any reasons to customize this component to hide the search taking into account that the same list is displayed?  It's not clear for me.

Reply:
I assume in this case we also can think that the list can be long as we are doing the same kind of request. Also do not forget about the sorting functionality. Now it exists in `SearchAnsSortPane` and I really wanted to reuse it.

Reply:
So, maybe, we can try to clarify requirements about the search field. And if it will be added in Q3 - we will not have to customize the current solution in this way. As I mentioned before, SearchAndSort with the hidden search functionality looks too complicated for me.

Reply:
I already asked a question about things but I am sticking to the opinion on leaving as is really even if it will be confirmed that we have such plans. If we have a search displayed now anyways it won't be going to work until Q3. I would rather have it hidden. I do not see a problem of having it hidden for at least for Q2. `SearchAndSortPane` now is a good way to accommodate all places, and personally I do not see much trouble of having search hidden as a part of `SearchAndSortPane`. It would be nice to start splitting IMO if we understand if there might some drastically another search.

Reply:
Okay. You have enough approves according to the DoD and can go ahead.

Reply:
Sounds good.